sources/System/Renderer/CameraSystem.cpp

```diff
@@ -59,8 +60,8 @@ namespace BBM
 		float lowerZDist = 0;
 
 		for (auto &[player, position, _] : this->_wal.getScene()->view<PositionComponent, TagComponent<Player>>()) {
-			if (!player.hasComponent<ControllableComponent>())
-				player.addComponent<ControllableComponent>();
+			if (!player.hasComponent<MovableComponent>())
```

Comment:
On another PR, I added a .disabled on the controllable component to handle this.
Your modification here could probably be removed when the merge is done.